examples/with-nextjs-and-clerk-auth/src/lib/monite-api/demo-data-generator/bin.ts

```diff
@@ -46,6 +48,115 @@ const commandWithEntityOptions = () =>
+      .description('Generate demo data for the existing Entity')
+      .action(async (args) => {
+        const token = await fetchTokenCLI(args);
+
+        const { id: newEntityId } = await createOrganizationEntity(
+          {
+            email: faker.internet.email(),
+            legal_name: faker.company.name(),
+          },
+          token
+        );
+
+        await generateDemoData(newEntityId, token);
+      })
+  )
+  .addCommand(
+    new Command()
+      .name('entity')
+      .description('Create a new Entity')
+      .option(
+        '--generate-demo-data',
+        'Generate demo data for the new entity',
+        true
+      )
+      .option('--client-id <ID>', 'The client_id to use', String)
+      .option('--client-secret <ID>', 'The client_secret to use', String)
+      .action(async (args) => {
+        const token = await fetchTokenCLI(args);
+
+        console.log(chalk.gray('Creating a new Entity with a few users...'));
+
+        const { id: newEntityId } = await createOrganizationEntity(
+          {
+            email: faker.internet.email(),
+            legal_name: faker.company.name(),
+          },
+          token
+        );
+
+        console.log(
+          chalk.greenBright(`🏢 Created new entity_id: "${newEntityId}"`)
+        );
+
+        console.log(
+          chalk.gray(
+            'Generating entity user role with permissions for the new entity...'
+          )
+        );
+
+        const entityRoles = await createEntityRoles(
+          newEntityId,
+          Object.keys(roles_default_permissions).filter(isExistingRole),
+          token
+        );
+
+        console.log(
+          chalk.greenBright(
+            `👥 Created new entity roles for the new entity_id:`
+          ),
+          chalk.bgGreen(`${newEntityId}`)
+        );
+
+        console.log(
+          chalk.gray('Generating entity users for the new entity...')
+        );
+        for (const [role, role_id] of Object.entries(entityRoles)) {
+          const { id } = await createEntityUser(
+            {
+              entity_id: newEntityId,
+              user: {
+                role_id,
+                login: faker.internet.userName(),
+                first_name: faker.person.firstName(),
+                last_name: faker.person.lastName(),
+                phone: faker.phone.number().slice(0, 15),
+              },
+            },
+            token
+          ).catch((error) => {
+            console.log(
+              chalk.red(
+                `Failed to create new entity_user with role: "${role}"`,
+                `${JSON.stringify(error)}`
+              )
+            );
+
+            return { id: null };
+          });
+
+          if (id)
+            console.log(
+              chalk.greenBright(
+                `👤 Created new entity_user_id: "${id}" with role: "${role}"`
+              )
+            );
+        }
+
+        if (args.generateDemoData) await generateDemoData(newEntityId, token);
```

Comment:
I'd prefer to use `{}` for all if / else statements for readability reasons. But it's a matter of code style and we don't have any rules so if you don't want to do it I have no permissions to force you

Reply:
If we find a sufficient reason to do so and enshrine it in a code-styled guide... In all other cases, it's a matter of taste.

Reply:
> it's a matter of taste

yep, yep, as I mentioned. No forcing to change it